sdk/spring/azure-spring-boot-test-aad/src/test/java/com/azure/test/aad/selenium/oauth2client/scopes/AADAccessTokenScopesIT.java

```diff
@@ -20,37 +16,36 @@
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.RestController;
 
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+
+import static com.azure.spring.test.EnvironmentVariable.AZURE_CLOUD_TYPE;
+import static com.azure.test.aad.selenium.AADSeleniumITHelper.createDefaultProperties;
+
 public class AADAccessTokenScopesIT {
 
     private AADSeleniumITHelper aadSeleniumITHelper;
 
     @Test
     public void testAccessTokenScopes() {
         Map<String, String> properties = createDefaultProperties();
-        properties.put(
-            "azure.activedirectory.authorization-clients.office.scopes",
-            "https://manage.office.com/ActivityFeed.Read, https://manage.office.com/ActivityFeed.ReadDlp, "
-                + "https://manage.office.com/ServiceHealth.Read");
-        properties.put(
-            "azure.activedirectory.authorization-clients.graph.scopes",
-            "https://graph.microsoft.com/User.Read, https://graph.microsoft.com/Directory.Read.All");
+
+        String graphBaseUrl = AzureCloudUrls.getGraphBaseUrl(AZURE_CLOUD_TYPE);
+        properties.put("azure.activedirectory.authorization-clients.graph.scopes",
+            graphBaseUrl + "User.Read, " + graphBaseUrl + "Directory.Read.All");
+
         aadSeleniumITHelper = new AADSeleniumITHelper(DumbApp.class, properties);
         aadSeleniumITHelper.logIn();
         String httpResponse = aadSeleniumITHelper.httpGet("accessTokenScopes/azure");
         Assert.assertTrue(httpResponse.contains("profile"));
-        Assert.assertTrue(httpResponse.contains("https://graph.microsoft.com/Directory.Read.All"));
-        Assert.assertTrue(httpResponse.contains("https://graph.microsoft.com/User.Read"));
+        Assert.assertTrue(httpResponse.contains("Directory.Read.All"));
+        Assert.assertTrue(httpResponse.contains("User.Read"));
 
         httpResponse = aadSeleniumITHelper.httpGet("accessTokenScopes/graph");
         Assert.assertTrue(httpResponse.contains("profile"));
-        Assert.assertTrue(httpResponse.contains("https://graph.microsoft.com/Directory.Read.All"));
-        Assert.assertTrue(httpResponse.contains("https://graph.microsoft.com/User.Read"));
-
-        httpResponse = aadSeleniumITHelper.httpGet("accessTokenScopes/office");
-        Assert.assertFalse(httpResponse.contains("profile"));
-        Assert.assertTrue(httpResponse.contains("https://manage.office.com/ActivityFeed.Read"));
-        Assert.assertTrue(httpResponse.contains("https://manage.office.com/ActivityFeed.ReadDlp"));
-        Assert.assertTrue(httpResponse.contains("https://manage.office.com/ServiceHealth.Read"));
```

Comment:
Add add scopes about "arm".
Refs: https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory-webapp/src/main/resources/application.yml#L4
Of course, no "on-demand".

And add test about "arm" scopes.